x/ibc/07-tendermint/types/client_state.go

```diff
@@ -114,7 +114,8 @@ func (cs ClientState) VerifyClientConsensusState(
 	proof commitmentexported.Proof,
 	consensusState clientexported.ConsensusState,
 ) error {
-	path, err := commitmenttypes.ApplyPrefix(prefix, ibctypes.ConsensusStatePath(counterpartyClientIdentifier, consensusHeight))
+	clientPrefixedPath := counterpartyClientIdentifier + "/" + ibctypes.ConsensusStatePath(consensusHeight)
```

Comment:
Simply appending here rather than using `ApplyPrefix` since the Prefix that gets used by all paths is simply `ibc`. Not sure if i should create a seperate Prefix object for `ClientConsensusState` and pass it in here or if I should continue hardcoding this append

If I create a prefix, I must have some code in the caller that does something like:

```
ibcPrefix :=  prefix.Bytes()
clientPrefix := commitment.NewPrefix(append(ibcPrefix, counterPartyClientID....)
// pass in clientPrefix to VerifyClientConsensusState, ibcPrefix into VerifyConnectionState
```

Reply:
I think from the perspective of the spec the full path just needs to be correct, I'm not sure what's considered more ergonomic in the SDK.